media-classification/src/main/resources/nl/vpro/domain/media/classification/ebu_ContentGenreCS.xml

```diff
@@ -131,6 +140,71 @@
           <Reference>urn:mis:genre:COMEDY</Reference>
           <ValidityFlag>1</ValidityFlag>
         </Term>
+        <Term termID="3.0.1.2.43">
+          <Name xml:lang="nl">Actie</Name>
+          <Reference>urn:tva:metadata:cs:2004:0245</Reference>
+          <ValidityFlag>1</ValidityFlag>
```

Comment:
wat in feite een beetj raar is dat we blijkbaar zo niet whatson volgen.

Waarom is onze termjd niet 3.0.1.2.45?   Het is zo gesuggereerd in de sheet, het lijkt toch nogal zot.

Reply:
Ik heb er een mailtje over gedaan naar Robbert-Jan